lib/wpc_proto/internal_modules/proto_config.c

```diff
@@ -15,8 +15,194 @@
 #include "logger.h"
 
 
+/** Structure to hold config from node */
+typedef struct sink_config
+{
+    uint16_t stack_profile;
+    uint16_t hw_magic;
+    uint16_t ac_limit_min;
+    uint16_t ac_limit_max;
+    uint16_t app_config_max_size;
+    uint16_t version[4];
+    uint8_t max_mtu;
+    uint8_t ch_range_min;
+    uint8_t ch_range_max;
+    uint8_t pdu_buffer_size;
+
+    bool CipherKeySet;
+    bool AuthenticationKeySet;
+    uint8_t StackStatus;
+    uint16_t ac_range_min_cur;  // 0 means unset ?
```

Comment:
Let's order them by size. Order in the list doesn't really matter.

Reply:
As we don't need to pack it, I'd prefer to leave it in a defined order, but ok, it's done.